infrastructure/terraform/components/reporting/scripts/create_iceberg_table.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Creates Iceberg table for reporting if it doesn't exist already
+
+ENV=${1:-"no_env"}
+
+if [[ ${ENV} == "no_env" ]]; then
+    echo "Environment name not provided as first argument"
+    exit 1
+fi
+
+table_exists=$(aws glue get-tables --database-name nhs-notify-${ENV}-reporting-database | jq 'any(.TableList[].Name == "'nhs_notify_${ENV}_item_status_iceberg'"; .)')
+
+if [[ ${table_exists} == "true" ]]; then
+    echo "Table already exists for this environment in the database, hence exiting gracefully"
+    exit 0
+fi
+
+
+account_id=""
+
+if [[ ${ENV} == "prod" ]]; then
+    account_id="211125615884"
+else
+    account_id="381492132479"
+fi
+
+execution_id=$( aws athena start-query-execution \
```

Comment:
DDL to create the table should be in a separate file, not embedded within the shell script.

Reply:
I have moved this to a sql file that I pass to the command now, hope thats ok